### ISSUE TITLE
CLI only

### ISSUE THREAD

**Issue by onyxcherry** [KaptainWutax/SeedCracker#204]
Hello,
how could I run this tool with **command line** only? Can I pass to your code (or a cli-wrapper) some data obtained myself (such as heigh of the end pillars etc.) and get a result? Where is an entrypoint?

Background: server uses Forge and other mods so I cannot use this and other Forge mods. CLI, as I not at all need GUI, would be sufficient probably.

**Comment by maxsupermanhd** [comment]
Agreed, @KaptainWutax it will be very very useful, sometimes auto-detection of shipwrecks is not working and ocean monuments too...
